File: EzDeal/API JAVA/src/main/java/com/myorg/ezdeal/models/Horario.java
package com.myorg.ezdeal.models;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.io.Serializable;
import java.time.LocalTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name="horario")
public class Horario implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="horario_id")
    private Integer id;

    @Column(name="dia")
    private String dia;

    @Column(name="hora_apertura")
    private LocalTime horaApertura;

    @Column(name="hora_cierre")
    private LocalTime horaCierre;

    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name="servicio_id")
    private Servicio servicio;


}

File: EzDeal/API .NET/EzDeal.Domain/Horario.cs
using System;
namespace EzDeal.Domain
{
    public class Horario
    {
        public long Id {get; set;}
        public string Dia {get; set;}
        public DateTime horaApertura {get; set;}
        public DateTime horaCierre {get; set;}
        
        public long ServicioId {get; set;}
        public Servicio Servicio {get; set;}
    }
}